BitwardenShared/UI/Auth/ProfileSwitcher/ProfileSwitcherView.swift

```diff
@@ -67,7 +61,7 @@ struct ProfileSwitcherView: View {
     private var backgroundView: some View {
         VStack {
             Asset.Colors.backgroundPrimary.swiftUIColor
-                .frame(height: abs(min(store.state.scrollOffset.y, 0)))
+                .frame(height: store.state.isVisible ? 0.01 : 0)
```

Comment:
was this needed?

Reply:
It doesn't seem to be necessary, at least on the iOS 18 device.